src/sync-controller.js

```diff
@@ -382,11 +383,12 @@ export default class SyncController extends videojs.EventTarget {
     }
   }
 
-  timestampOffsetForTimeline(timeline) {
-    if (typeof this.timelines[timeline] === 'undefined') {
+  timestampOffsetForSegment(segmentInfo) {
+    if (segmentInfo.timeline > 0 &&
```

Comment:
You might want to use intermediary variables for clarity. I'm not sure what these checks are doing.

```
const somethingDescriptive = (!segmentInfo.startOfSegment || segmentInfo.startOfSegment > this.lastBufferedSegmentTimestamp);

if (segmentInfo.timeline > 0 && somethingDescriptive) {
  return null;
}

Reply:
Done